test_runner/src/main/kotlin/ftl/config/common/CommonFlankConfig.kt

```diff
@@ -148,6 +149,9 @@ data class CommonFlankConfig @JsonIgnore constructor(
     @set:JsonProperty("default-test-time")
     var defaultTestTime: Double? by data
 
+    @set:JsonProperty("default-class-test-time")
+    var defaultClassTestTime: Double? by data
```

Comment:
Does `keys` list need this new value to be able to check by flank doctor? `defaultTestTime` is also missing in `key` list.